docs/subnets/register-validate-mine.md

```diff
@@ -54,23 +63,28 @@ You stake by attaching TAO to your hotkey. Attaching TAO to your hotkey can be a
 
 #### Stake your own TAO
 
-By staking your own TAO funds to your hotkey that holds the UID in the subnet where you want to validate.
+You can do this by staking your own TAO funds to your hotkey, which holds the UID in the subnet where you want to validate.
+
+```bash
+#Stake funds to your hotkey account within the subnet.
+btcli stake add --wallet.name <my_coldkey> --wallet.hotkey <my_hotkey>
+```
+Example:
 ```bash
-# Stake funds to your hotkey account within the subnet.
-btcli stake add
-    --wallet.name YOUR_WALLET_NAME
-    --wallet.hotkey YOUR_HOTKEY_NAME
+btcli stake add --wallet.name test-coldkey --wallet.hotkey test-hotkey
 ```
 
 #### Attract delegated stake 
 
-By attracting delegated stake from the nominators. For this you must first nominate your hotkey as a delegate and then advertising your hotkey. The nominators can then delegate their TAO to your hotkey. 
+Attract delegated stake by attracting delegated stake from the nominators. For this, you must first nominate your hotkey as a delegate and then advertise it. The nominators can then delegate their TAO to your hotkey.
```

Comment:
Let's rewrite 1st sentence like this:

"You can also increase your stake by attracting delegated stake from the nominators."

Let's clarify the second sentence by replacing "it". Generally speaking, the word "it" very often confuses me, because I don't know immediately what this "it" refers to. Suggest rewrite like this:

"For this, you must first nominate your hotkey as a delegate and then advertise this hotkey."